tests/generate_code.bat

```diff
@@ -15,6 +15,7 @@
 set buildtype=Release
 if "%1"=="-b" set buildtype=%2
 
+..\%buildtype%\flatc.exe --cpp nestedtable_test.fbs
```

Comment:
Lets not add a new schema for this, please test with the existing schema as best you can.